packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.test.ts

```diff
@@ -31,83 +31,83 @@ describe('generateCognitoStackResources', () => {
     const testApp = new cdk.App();
     const cognitoStack = new AmplifyAuthCognitoStack(testApp, 'CognitoPreSignUpTriggerTest', { synthesizer: new AuthStackSynthesizer() });
     const props : CognitoStackOptions = {
-        "identityPoolName": "issue96802f106de3_identitypool_2f106de3",
-        "allowUnauthenticatedIdentities": false,
-        "resourceNameTruncated": "issue92f106de3",
-        "userPoolName": "issue96802f106de3_userpool_2f106de3",
-        "autoVerifiedAttributes": [
-          "email"
+      identityPoolName: 'issue96802f106de3_identitypool_2f106de3',
+      allowUnauthenticatedIdentities: false,
+      resourceNameTruncated: 'issue92f106de3',
+      userPoolName: 'issue96802f106de3_userpool_2f106de3',
+      autoVerifiedAttributes: [
+        'email',
+      ],
+      mfaConfiguration: 'OFF',
+      mfaTypes: [
+        'SMS Text Message',
+      ],
+      smsAuthenticationMessage: 'Your authentication code is {####}',
+      smsVerificationMessage: 'Your verification code is {####}',
+      emailVerificationSubject: 'Your verification code',
+      emailVerificationMessage: 'Your verification code is {####}',
+      passwordPolicyMinLength: 8,
+      passwordPolicyCharacters: [],
+      requiredAttributes: [
+        'email',
+      ],
+      aliasAttributes: [],
+      userpoolClientGenerateSecret: false,
+      userpoolClientRefreshTokenValidity: 30,
+      userpoolClientWriteAttributes: [
+        'email',
+      ],
+      userpoolClientReadAttributes: [
+        'email',
+      ],
+      userpoolClientLambdaRole: 'issue92f106de3_userpoolclient_lambda_role',
+      userpoolClientSetAttributes: false,
+      sharedId: '2f106de3',
+      resourceName: 'issue96802f106de32f106de3',
+      authSelections: 'identityPoolAndUserPool',
+      useDefault: 'manual',
+      thirdPartyAuth: false,
+      userPoolGroups: false,
+      adminQueries: false,
+      triggers: {
+        PreSignup: [
+          'custom',
         ],
-        "mfaConfiguration": "OFF",
-        "mfaTypes": [
-          "SMS Text Message"
+      },
```

Comment:
It's kind of hard to diff this. Is this a formatting change or is there a change in the structure?
If the latter what has changed?